Source/Resources/en.lproj/Localizable.strings

```diff
@@ -24,6 +24,7 @@
 "PaymentHeaderSubtitle" = "Visa, Mastercard and American Express accepted.";
 "CardNumber" = "Card number";
 "CardNumberErrorMessage" = "Please enter a valid card number";
+"Pay" = "Pay";
```

Comment:
There's no amount at all for default screen.

Reply:
Note : This amount will be passed by merchants if they want the amount to be visible after the Pay Text as per the design. 

@alex-ioja-yang-cko the %@ will be replaced by any string passed by merchant. 

However the other problem I see here is there is no way for us to validate this value. Merchant can create a token, call the /payments end point ( for 500£ ) and on the screen show just 5£. In this case it will be merchants responsibility they pass right values to payments end point and what they show on screen. @patrick-hoban-cko are we fine with that behaviour ?

Reply:
Should we then remove the number from the default? The user of SDK can pass the title as they wish (with value, currency, however they want it) and our default will remove this false expectation.

We probably should add the number from the demo app (CustomStyle), not from SDK then

Reply:
Yes, we should add the number from the demo app, not from the SDK.
 
That would imitate how a merchant actually uses this - they pass the value into the string in order to show the correct number in the Pay button. We don't define that for them of course, as we don't know the value of the basket.